Controller/RegistrationController.php

```diff
@@ -96,7 +96,7 @@ public function registerAction(Request $request)
     /**
      * Tell the user to check his email provider.
      */
-    public function checkEmailAction()
+    public function checkEmailAction(Request $request)
```

Comment:
Apologies, I thought you meant something else. With `Request $request = null` the page seems to work. This is not an exhaustive test though.

However, I believe the recommended signature in symfony 3.0+ is `Request $request`. Why would the request be null? Are you concerned about backward compatibility with existing code? Why would that action be invoked with a null request?